data/hai/hai reveal 4 middle.txt

```diff
@@ -76,20 +77,28 @@ mission "Hai Reveal [A02] Symbol Identification"
 			`	"Greetings Captain <last>, my name is Teeneep Goot'r. I'm a social advocate among the Hai, and I'm trying to help your friends with a problem. As you know, they were kidnapped by humans. A few other Hai and human victims are here; they were mostly rescued by other kind merchants in human space, but around half of those here are my staff."`
 			`	"My organization has been researching the issue of disappearing Hai ever since some humans also began disappearing here under similar circumstances, and we've discovered some patterns to the disappearances. Perhaps more pertinently though, since we've found those patterns we've been able to reach some of the victims more quickly upon their return and even rescue a couple ourselves. However, those Hai that you rescued and several others, too, have a very strange commonality. These kidnap victims are all experiencing mild cadmium poisoning."`
 			choice
-				`	"What patterns have you noticed in the kidnappings?"`
-					goto patterns
-				`	"Tell me about the cadmium poisoning."`
-					goto cadmium
-				`	"How long have the kidnappings been happening?"`
-					goto kidnappings
+				`	"Before we discuss that, can you tell me if the Hai government is arming pirates?"`
+					goto direct
+				`	"It sounds like something I can help with, but there's something I need from you first."`
+					goto infotrade
+			label infotrade
+			`	She sounds familiar with the information trade, as she says, "What do you need?"`
```

Comment:
```suggestion
			`	She seems quite familiar with the practice of trading information, as she simply says, "What do you need?"`
```

Another slight tone adjustment, a little less formal acknowledgment of the arrangement but also more assumptive about Teeneep's usage of it.